### PR TITLE
UKVIET-9: Upgrade to HOF v19 with pipeline upgrades

### PR DESCRIPTION
**Pull request by Alex-Swann**

### CODE REVIEW

File: clean_up.sh [hunk 1]
```diff
@@ -0,0 +1,35 @@
+#! /bin/bash
+set -e
+
+export IGNORE_NETPOL=("acp-deny-all")
+export IGNORE_CONFIGMAP=("bundle")
+export IGNORE_SECRET=("file-vault-cert" "firearms-notprod-s3" "icasework" "keycloak" "keycloak-form" "postcode-auth" "redis" "s3-bucket" "ses" "session-secret")
```
Comment (MaxOSullivan) [inline clean_up.sh:6]: Update names

File: hof.settings.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  "appName": "UKVI Ending of Tenancy",
+  "theme": "govUK",
+  "behaviours": [
+    "hof/components/clear-session"
+  ],
+  "routes": [
+    "./apps/end-tenancy"
+  ],
+  "session": {
+    "name": "ukvi-end-tenancy.hof.sid"
```
Comment (MaxOSullivan) [inline hof.settings.json:11]: On the /cookies page the session ID is listed as hod.sid - one will need to be updated

**Review by MaxOSullivan — COMMENTED** [review]
2 comments

File: hof.settings.json (hunk 2 above)
Comment (Alex-Swann) [reply hof.settings.json:11]: cookies page is updated with this now on the latest HOF 👍

File: clean_up.sh (hunk 1 above)
Comment (Alex-Swann) [reply clean_up.sh]: this has been refactored now and isn't included

**Pull request merged**